test/integration/backward_compatible/map/MapProxyTest.js

```diff
@@ -110,31 +110,31 @@ describe('MapProxyTest', function () {
             });
 
             it('put with ttl removes value after ttl', async function () {
-                await map.put('key10', 'val10', 1000);
+                await map.put('key10', 'val10', 2000);
```

Comment:
I think the TTL here is too low. It's 20 seconds in the Go client. Similar for other related code in this PR.

Reply:
2 seconds did not cause a problem in master for a very long time. Increasing this will lead to longer tests. What about increasing again if a fail happens?

Reply:
I have made tts 20 in all backports and opened a new pr for master.